docs/troubleshooting-guide/error-message.md

```diff
@@ -0,0 +1,34 @@
+# What type of error message do you see?
+
+## API error messages
+
+- The error message appears in red text when you load a dashboard or run a question.
+- The error message contains a three-digit API error code, such as `400` or `404`.
+- If your error message contains part of your SQL query, go to SQL error messages below.
+
+### What to do:
+
+- Search for your error code or ask the community on [Discourse][discourse].
+
+## [SQL error messages][sql-error-message]
+
+- The error message appears in red text when you run a question that uses the [SQL editor][sql-editor].
+- The error message contains part of your SQL query, such as a column or table name.
+- The error message may also contain a three-digit API error code, such as `400` or `404`.
+
+### What to do:
+
+- Go to [Troubleshooting SQL error messages][sql-error-message].
+- If you can’t solve your problem using the troubleshooting guides, search for your error message or ask the community on [Discourse][discourse].
+
+## Metabase error messages
```

Comment:
Would images of each of these different error types be useful?

Reply:
Yeah, good call. I imagine I'll be updating these pages frequently, will generate some errors for both examples and images on the next go-around.